about/glossary.rst

```diff
@@ -22,13 +22,32 @@ Glossary
       Amazon Web Services: Cloud computing infrastructure provided by Amazon 
       (United States).   
    
+   BRDF
+      “Bidirectional reflectance distribution function” is a theoretical concept that describes the 
```

Comment:
try ``Bidirectional reflectance distribution function`` with doube `

Reply:
I think quotes are ok here since it's not for formatting but indicating that it's what BRDF stands for.